src/App.tsx

```diff
@@ -76,18 +65,14 @@ export const App: React.FC = () => {
                   </div>
                 )}
 
-                {author && loaded && !hasError && posts.length === 0 && (
+                {author && !loading && !error && posts.length === 0 && (
```

Comment:
it makes sense move these conditions to a var 
```suggestion
                {author && !loading && !error && !posts.length && (
```